app/controllers/candidate_interface/degrees/degree/destroy_controller.rb

```diff
@@ -7,6 +7,9 @@ class DestroyController < BaseController
 
         def confirm_destroy
           @degree = current_degree
+          if @degree.nil?
```

Comment:
Is this line needed? Can we not redirect always idenpendent if is nil or not? Or use the `destroyed?` check? ([reference](https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-destroyed-3F))

Reply:
doesn't `destroyed?` still error as it's nilclass? I think we need the line, otherwise the redirect will prevent the destroy controller action from being submitted. When there are no degrees they can't get to the confirm destroy page anyway, its only to deal with the browser back, happy to discuss